<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Warcoin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="layoutGry">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <zorder>peerList</zorder>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nrTuryLbl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Numer tury: ?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nowaTuraBtn">
        <property name="text">
         <string>Nowa tura</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="nawiazanePolaczeniaLbl">
        <property name="text">
         <string>Nawiązane połączenia:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="peerList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rozlaczBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Rozłącz</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="adresIpLayout">
        <item>
         <widget class="QLabel" name="adresIpLbl">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Adres IP:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="adresIpEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0.0.0.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="numerPortuLayout">
        <item>
         <widget class="QLabel" name="numerPortuLbl">
          <property name="text">
           <string>Numer portu:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="numerPortuEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="podlaczSieBtn">
        <property name="text">
         <string>Podłącz się</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
